app/uk/gov/hmrc/thirdpartydeveloperfrontend/connectors/DeskproConnector.scala

```diff
@@ -37,7 +38,7 @@ class DeskproConnector @Inject() (http: HttpClient, config: ApplicationConfig, m
   lazy val serviceBaseUrl: String = config.deskproUrl
   val api                         = API("deskpro")
 
-  def createTicket(deskproTicket: DeskproTicket)(implicit hc: HeaderCarrier): Future[TicketResult] = metrics.record(api) {
+  def createTicket(id: String, deskproTicket: DeskproTicket)(implicit hc: HeaderCarrier): Future[TicketResult] = metrics.record(api) {
```

Comment:
It may be better to have a createTicket method for each id we could pass in.
These could then delegate to a private method which takes a log message specific to the id type.

createTicket(userId...)
createTicket(riVerification...)

Each of these could build their own error message specific to the id type.
This means we wouldn't have the overly general "Deskpro ticket creation failed for userId / ResponsibleIndividualVerification:" message